detections/endpoint/detect_mshta_url_in_command_line.yml

```diff
@@ -78,8 +78,7 @@ tags:
   - Compromised Windows Host
   - Lumma Stealer
   - Living Off The Land
-  - Suspicious MSHTA Activity
-  - XWorm
```

Comment:
we just added  `XWorm`  in this release v5.6.0 and looks like it got removed in this change. 

also, Compromised Windows Host is listed twice in the analytic story